packages/models/src/errors.ts

```diff
@@ -105,6 +105,8 @@ const errorCodes = {
   missingRSAKey: "9996",
   missingKafkaMessageData: "9997",
   kafkaMessageProcessError: "9998",
+  // This error code is temporary and should be removed when conversion is deprecated
+  eventV1ConversionError: "9999",
```

Comment:
Maybe we don't need to have this code in shared models but we can keep it confined as a code only specific to notifier-seeder, like done for the error codes in file-manager / queue-manager for example